src/main/java/com/timeplus/kafkaconnect/TimeplusSinkTask.java

```diff
@@ -133,20 +141,16 @@ private String getCreateRawPayload(String stream) {
         JSONObject payload = new JSONObject();
         payload.put("name", stream);
 
-        JSONObject col = new JSONObject();
-        col.put("name", "raw");
-        col.put("type", "string");
-
         JSONArray cols = new JSONArray();
```

Comment:
It looks this `cols` array is always the same, does it need to be created everytime?